Library/Homebrew/cmd/analytics.rb

```diff
@@ -0,0 +1,44 @@
+#:  * `analytics` [`state`]:
+#:    Display analytics state.
+#:
+#:  * `analytics` `on`|`off`:
```

Comment:
This should be

```
#:  * `analytics` (`on`|`off`):
```

to be consistent with other documentation we have where there's a list of mandatory alternatives.